mkdocs.yml

```diff
@@ -177,7 +177,9 @@ nav:
   - developer-guide/releasing.md
   - developer-guide/site.md
   - developer-guide/static-code-analysis.md
-  - developer-guide/ui-extensions.md
+  - Extensions:
```

Comment:
Will this change the URL of the old extensions page? If so, we should include a stub page to link to the new location.

Reply:
done